core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala

```diff
@@ -208,7 +208,6 @@ class HistoryServerSuite extends SparkFunSuite with BeforeAndAfter with Matchers
         new File(expRoot, HistoryServerSuite.sanitizePath(name) + "_expectation.json")),
         StandardCharsets.UTF_8)
       // compare the ASTs so formatting differences don't cause failures
-      import org.json4s._
```

Comment:
It seems that this is not the only occurrence to be removed.

Reply:
BTW, this become unused after M16?

Reply:
yes, with M16 it is throwing Unused import error on org.json4s._

Reply:
Due to binary incompatibility with M16, i changed to M11